src/Operation/Count.php

```diff
@@ -162,6 +162,39 @@ public function execute(Server $server)
         return (integer) $result->n;
     }
 
+    /**
+     * Explain the operation.
+     *
+     * @see Explainable::explain()
+     * @param Server $server
+     * @param $command
+     * @param array $options
+     * @return array|object
+     * @throws DriverRuntimeException for other driver errors (e.g. connection errors)
+     */
+    public function explain($server, $command, $options = [])
+    {
+        if ( ! isset($options['verbosity'])) {
+            $options['verbosity'] = 'allPlansExecution';
+        }
+
+        $cmd = new Command(['explain' => ['count' => $this->collectionName, 'query' => $command], 'verbosity' => $options['verbosity']]);
+
+        if (empty($command)) {
+            $cmd = new Command(['explain' => ['count' => $this->collectionName], 'verbosity' => $options['verbosity']]);
+        }
+
+        $result = $server->executeCommand($this->databaseName, $cmd);
+
+        $resultArray = get_object_vars($result->toArray()[0]); // cast $result to array
+
+        if ($options['verbosity'] === 'queryPlanner') {
+            return ['queryPlanner' => $resultArray['queryPlanner']];
```

Comment:
Is there a reason for not returning the result document as-is? There are a few other operation classes where we do so (e.g. CreateCollection and drop operations). There may be some extra fields, such as `ok` and gossiped cluster time, but I don't think we need to insulate users from that. We can always look at crafting an ExplainResult class down the line if the raw result document is too noisy.